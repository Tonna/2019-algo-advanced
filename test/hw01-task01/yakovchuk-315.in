25
1 1 1 1 2 4 5 1 1 2 4 2 10 4 1 1 2 16 6 6 16 3 20 9
10
13 2
14 6
2 8
15 1
21 6
24 4
2 2
7 9
8 2
8 1
