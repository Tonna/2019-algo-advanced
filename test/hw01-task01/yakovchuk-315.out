4
16
15
4
18
15
4
17
6
5
